app/controllers/follower_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FollowerController < ApplicationController
+  def show
```

Comment:
chuyen the nay cung dc, nhưng anh muốn chú viết gọp thành 1 action cho cả flower và flowing luôn

Reply:
mình vẫn chưa hiểu đoạn này lắm :thinking: